src/main/drivers/rx/rx_sx1280.c

```diff
@@ -208,6 +208,7 @@ bool sx1280Init(IO_t resetPin, IO_t busyPin)
 
     if (busyPin) {
         IOInit(busyPin, OWNER_RX_SPI_EXPRESSLRS_BUSY, 0);
+        IOConfigGPIO(busyPin, IOCFG_IN_FLOATING);
```

Comment:
Is the same needed in `rxSpiDeviceInit()` for `extiPin`?

Reply:
no, the EXTIConfig does it IIRC, but BUSY pin is used as a GPIO *INPUT* prior to being used for EXTI and the GPIO mode was ANALOG, not INPUT due to the MCU defaults, as per the description in #11451.